patches/minecraft/net/minecraft/entity/EntityLiving.java.patch

```diff
@@ -31,11 +39,20 @@
          else
          {
              Entity entity = this.field_70170_p.func_72890_a(this, -1.0D);
-@@ -845,7 +858,6 @@
+@@ -845,7 +859,6 @@
          return !this.field_70170_p.func_72953_d(this.func_174813_aQ()) && this.field_70170_p.func_184144_a(this, this.func_174813_aQ()).isEmpty() && this.field_70170_p.func_72917_a(this.func_174813_aQ(), this);
      }
  
 -    @SideOnly(Side.CLIENT)
      public float func_70603_bj()
      {
          return 1.0F;
+@@ -997,7 +1010,7 @@
+ 
+     public static EntityEquipmentSlot func_184640_d(ItemStack p_184640_0_)
+     {
+-        return p_184640_0_.func_77973_b() != Item.func_150898_a(Blocks.field_150423_aK) && p_184640_0_.func_77973_b() != Items.field_151144_bL ? (p_184640_0_.func_77973_b() instanceof ItemArmor ? ((ItemArmor)p_184640_0_.func_77973_b()).field_77881_a : (p_184640_0_.func_77973_b() == Items.field_185160_cR ? EntityEquipmentSlot.CHEST : (p_184640_0_.func_77973_b() == Items.field_185159_cQ ? EntityEquipmentSlot.OFFHAND : EntityEquipmentSlot.MAINHAND))) : EntityEquipmentSlot.HEAD;
++        return p_184640_0_.func_77973_b() != Item.func_150898_a(Blocks.field_150423_aK) && p_184640_0_.func_77973_b() != Items.field_151144_bL ? (p_184640_0_.func_77973_b() instanceof ItemArmor ? ((ItemArmor)p_184640_0_.func_77973_b()).field_77881_a : (p_184640_0_.func_77973_b() == Items.field_185160_cR ? EntityEquipmentSlot.CHEST : (p_184640_0_.func_77973_b() instanceof ItemShield ? EntityEquipmentSlot.OFFHAND : EntityEquipmentSlot.MAINHAND))) : EntityEquipmentSlot.HEAD;
```

Comment:
Instead of patching that ugly ternary decompilation mess, just add `if (stack.getItem() instanceof ItemShield) return OFFHAND;`